src/Surfnet/StepupRa/RaBundle/Resources/views/Vetting/Sms/provePossession.html.twig

```diff
@@ -20,10 +20,36 @@
 
     <hr>
 
-    {{ form(form) }}
+    {{ form_start(form) }}
+    {{ form_errors(form) }}
+    <div class="row">
+        <div class="col-md-4">
+            {{ form_label(form.challenge, null, {'label_attr': {'class': 'pull-right'}}) }}
+        </div>
+        <div class="col-md-8">
+            {{ form_widget(form.challenge) }}
+        </div>
+    </div>
+    <div class="row">&nbsp;</div>
+    <div class="row offset-top">
+        <div class="col-md-1">
+            &nbsp;
+        </div>
+        <div class="col-md-5">
+            {{ form_row(form.resendChallenge) }}
+        </div>
+        <div class="col-md-3">
+            <button name="ra_verify_phone_number[cancel]" class="btn btn-danger pull-right" formnovalidate="formnovalidate">
```

Comment:
Oh, and is it possible to get the `button[name]` from the form element so this is less fragile?